trajectory_node:
  ros__parameters:
    # Frames & Topics
    frame_id: "world"
    veh_state_topic: "/slam/vehicle_state"
    map_topic: "/slam/map"
    path_topic: "/trajectory/middle_line"
    mission_topic: "/master/mission"
    markers_topic: "/traj/markers"

    # Sampling for resampled path (m)
    target_ds: 0.5

    # valid: trackdrive | skidpad | acceleration | inspection
    mission_type: "trackdrive"

    # Optional Planner-Flag
    experimental_perf: false

    # Replan-strategy
    replan_on_state: true
    min_distance_replan: 0.5     # [m]
    min_yaw_replan: 0.10         # [rad]
    min_period_replan: 0.2       # [s]
